.gitlab-ci.yml

```diff
@@ -67,6 +67,21 @@ build-linux:
   tags:
     - linux-docker
 
+build-snap-i386:
+  <<:                              *build-linux
+  variables:
+    CARGO_TARGET:                  i686-unknown-linux-gnu
+
+build-snap-arm64:
+  <<:                              *build-linux
+  variables:
+    CARGO_TARGET:                  aarch64-unknown-linux-gnu
+
+build-snap-armhf:
+  <<:                              *build-linux
+  variables:
+    CARGO_TARGET:                  armv7-unknown-linux-gnueabihf
```

Comment:
If these changes will be merged, would be smart to preinstall these targets in the new Dockerfile. And push it to production right away.